Formula/aws-auth.rb

```diff
@@ -0,0 +1,29 @@
+require "language/node"
+
+class AwsAuth < Formula
+  desc "Allows you to programmatically authenticate into AWS accounts through IAM roles"
+  homepage "https://github.com/iamarkadyt/aws-auth#readme"
+  url "https://registry.npmjs.org/@iamarkadyt/aws-auth/-/aws-auth-2.1.2.tgz"
+  sha256 "570f767500ed75aa3855338914a4bd2b1ddc030c5b9574fc257b1c59150380c4"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink libexec.glob("bin/*")
+
+    # Replace universal binaries with native slices.
+    deuniversalize_machos
```

Comment:
Is there no way to only install the native binaries?

Reply:
I removed the lines since there are no native binaries